crates/astria-composer/src/searcher/mod.rs

```diff
@@ -201,23 +194,37 @@ impl Searcher {
         loop {
             select!(
                 // serialize and sign sequencer tx for incoming pending rollup txs
-                Some(rollup_tx) = self.new_transactions_rx.recv() => self.bundle_pending_tx(rollup_tx),
+                Some(rollup_tx) = self.new_transactions_rx.recv() => self.serialize_pending_tx(rollup_tx),
```

Comment:
I did this initially but then decided against it because then serialization blocks on reaping other transactions from the geth mempool. Happy to add this in tho.